pkg/hive/internal/text.go

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package internal
+
+import "strings"
+
+// LeftPad pads all lines in string 's' with 'n' spaces.
+func LeftPad(s string, n int) string {
```

Comment:
You can pad left with Printf/Sprintf, no need to re-implement
```go
fmt.Sprintf("%*s", width, str)
```

Reply:
doh, had forgotten that, thanks!